package.json

```
@@ -30,7 +30,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick"
+      "pre-commit": "pretty-quick && jest --onlyChanged"
```

```suggestion
      "pre-commit": "pretty-quick"
```

I wouldn't run them on the git commit, people might want to commit work in progress stuff.

Please add a `test` script for now